README.md

```diff
@@ -49,7 +49,10 @@ example_config for an example on how to do that.
 In order to build a compressed image of the server using docker, use the "make  
 docker" command. This shrunken image is intended to be used to run the  
 webserver on cloud hosts such as Amazon Web Services. Docker must be installed  
-prior to making or using the image.
+prior to making or using the image. In addition, in order to deploy your most 
+recent docker image to the AWS instance, run "make deploy", which shrinks the 
```

Comment:
"make docker" creates the shrunken image
"make deploy" saves the image to a file so it can be transferred.

I would change "shrinks" to "saves" on line 53